server/src/main/java/swim/tutorial/UnitAgent.java

```diff
@@ -12,14 +12,36 @@
 import java.util.Iterator;
 
 public class UnitAgent extends AbstractAgent {
+		
+  // TODO: complete the stats Value Lane
+  // @SwimLane("stats")
+	
+  // HINT: Use the valueLane() method to instantiate the lane
+  // HINT: Use the .didSet() lifecycle callback to log a message showing updates to stats
+	
```

Comment:
Adding some hints for intermediary statistic lanes here-ish would be nice.

Reply:
Hopefully the following comments (lines 23-30) better explain one's options for how to implement lanes. (Let me know if the wording can be made clearer.)